packages/yoga/src/List/native/ListItem.jsx

```diff
@@ -20,26 +22,55 @@ const StyledView = styled.View(
       : ''
   }
 
-    padding:
-      ${list.listItem.padding.top}px
-      ${list.listItem.padding.right}px
-      ${list.listItem.padding.bottom}px
-      ${list.listItem.padding.left}px;
+  ${
+    small
+      ? `
+          padding:
+            ${list.listItem.small.padding.top}px
+            ${list.listItem.small.padding.right}px
+            ${list.listItem.small.padding.bottom}px
+            ${list.listItem.small.padding.left}px;
+        `
+      : `
+          padding:
+            ${list.listItem.padding.top}px
+            ${list.listItem.padding.right}px
+            ${list.listItem.padding.bottom}px
+            ${list.listItem.padding.left}px;
+        `
+  }
   `,
 );
 
-const ListItem = ({ divided, ...rest }) => (
-  <StyledView divided={divided} {...rest} />
-);
+const ListItem = ({ theme, small, divided, onPress, ...rest }) => {
```

Comment:
hey Yuki, we have a non-documented (sorry sorry sorry, my fault) convention that components use
we agreed to destructure the theme to the declared component level, so we avoid a lot of dot property accessors
what do you think?

Reply:
Yes, sure! Updated!